scripts/appleseedMaya/renderGlobals.py

```diff
@@ -517,7 +517,7 @@ def create(self):
                             label="Max Samples",
                             columnWidth=(3, 160),
                             columnAttach=(1, "right", 4),
-                            minValue=16,
+                            minValue=1,
```

Comment:
Better!

Noticing that it's set to 0 in appleseed.studio, which feels wrong. Will check with @oktomus.